docs/_templates/autosummary/class.rst

```diff
@@ -0,0 +1,31 @@
+{% extends "!autosummary/class.rst" %}
+
+{% block methods %}
+{% if methods %}
+.. rubric:: {{ _('Methods') }}
+
+.. autosummary::
+    :toctree:
```

Comment:
We need to modify `autosummary/class.rst` template to include `:toctree:` directive to make methods pages be accessible via side panel.

Reply:
irrelevant